packages/design-tokens/stories/BrandColors.stories.tsx

```diff
@@ -26,9 +26,11 @@ type Story = StoryObj<typeof ColorSwatchGroup>;
 export const Figma: Story = {
   render: () => {
     const { brandColor } = useJsonColor();
-    console.log('brandColor JSON', brandColor);
     return <ColorSwatchGroup swatchData={brandColor} />;
   },
+  parameters: {
+    colorScheme: 'light',
+  },
```

Comment:
This locks the Figma story to light mode otherwise we get dual and it doesn't look right.

<img width="1497" alt="Screenshot 2024-11-21 at 6 06 21 PM" src="https://github.com/user-attachments/assets/0c465543-c88e-4915-bf9e-72241e30531e">